backend/app/src/test/kotlin/io/tolgee/api/v2/controllers/translations/v2TranslationsController/TranslationsControllerViewTest.kt

```diff
@@ -245,6 +246,24 @@ class TranslationsControllerViewTest : ProjectAuthControllerTest("/v2/projects/"
       }
   }
 
+  @ProjectJWTAuthTestMethod
+  @Test
+  fun `returns keys filtered by provided tags`() {
+    testData.generateLotOfData()
+    testDataService.saveTestData(testData.root)
+    userAccount = testData.user
+    val expectedKeys = IntRange(1, 9).map { "key 0$it" }
+    performProjectAuthGet("/translations/en,de?filterTag=${FIRST_9_KEYS_TAGS}&filterTag=unknownTag")
```

Comment:
I would at least put the constant into the TranslationsTestData class and not to the top level.

Reply:
Now deleted.